src/main/java/org/osc/sdk/controller/DefaultInspectionPort.java

```diff
@@ -60,7 +67,28 @@ public String getParentId() {
 
     @Override
     public String toString() {
-        return "DefaultInspectionPort [ingressPort=" + this.ingressPort + ", egressPort=" + this.egressPort + "]";
+        return "DefaultInspectionPort [ingressPort=" + this.ingressPort + ", egressPort=" + this.egressPort + ", element="
+                + this.element + "]";
     }
 
+    private static class ElementImpl implements Element {
```

Comment:
@dmitryintel Element Impl needs a toString method as well or printing out this.element will not give the id. You can address in the next checking